hw/ip/otbn/rtl/otbn_model.cc

```diff
@@ -0,0 +1,141 @@
+#include "svdpi.h"
```

Comment:
Is this header the header that this TU is implementing? If not, it should co after the system includes.